diesel/Cargo.toml

```diff
@@ -17,6 +17,7 @@ byteorder = "0.3.*"
 quickcheck = { version = "0.2.25", optional = true }
 chrono = { version = "^0.2.17", optional = true }
 uuid = { version = "^0.2.0", optional = true, features = ["use_std"] }
+log = { version = "0.3", optional = true }
```

Comment:
If we do go forward with using the log crate for this, I wonder if it would make sense for the other connections just to call those macros directly instead of a separate struct.